packages/generators/init-generator.ts

```diff
@@ -133,6 +133,8 @@ export default class InitGenerator extends Generator {
 				chunkFilename: "'[name].[chunkhash].js'",
 				filename: "'[name].[chunkhash].js'"
 			};
+		} else {
```

Comment:
remove this

Reply:
but what about the output directory then ?....if it is specified by the user then shouldn't it be added ?

Reply:
If not specified we are setting a default one in webpack-cli no need to add it to config

Reply:
What about when its specified?

Reply:
I think we're setting that prop further up the Lines.

Reply:
I guess it was not adding any, ok will test this particular one more time and will get back here

Reply:
@evenstensberg I checked with when specified the output directory, `output.path` not reflecting in the config file, its simply giving the chunk file support.

Reply:
I'm working on these defaults next so leave them to me, need to check in with some folks and speak to various of people to set the correct defaults

Reply:
sure then!!

Reply:
done